src/constants/categories.js

```diff
@@ -16,7 +16,7 @@ export default [
   {
     id: 'infrastructure',
     colorName: COLOR_NAMES.INFRASTRUCTURE,
-    name: 'Infrastructure & Other Things',
+    name: 'Infrastructure',
```

Comment:
I think this will actually need to be a `navName` property since the page title is still "Infrastructure & Other Things". Same as "Livability & sustainability" and "Liveability" for the tab name.